lib/src/main/java/com/orange/ods/compose/theme/OdsTypography.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.compose.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.orange.ods.R
+
+private val Roboto = FontFamily(
+    Font(R.font.roboto_regular, FontWeight.Normal),
+    Font(R.font.roboto_bold, FontWeight.Bold)
```

Comment:
I'm not sure you need this.
https://stackoverflow.com/a/13329907/4070971
https://stackoverflow.com/questions/69220102/sansserif-fontfamily-in-jetpack-compose